front/cypress/e2e/content_builder/components/about_component.cy.ts

```diff
@@ -40,13 +38,10 @@ describe.skip('Content builder About component', () => {
   });
 
   it('handles About component correctly', () => {
-    cy.visit(`/admin/content-builder/projects/${projectId}/description`);
-    cy.get('#e2e-draggable-single-column').dragAndDrop(
-      '#e2e-content-builder-frame',
-      {
-        position: 'inside',
-      }
+    cy.intercept('**/content_builder_layouts/project_description/upsert').as(
```

Comment:
We are intercepting the request to update the content builder...

Reply:
Just to confirm - this intercept code needs to be run in each "it" test right? We aren't able (or maybe it's bad practice?) to do this once somewhere where the whole file can then see it?

Reply:
Yep, I think so.. Maybe we can abstract it in a custom command in the future.